frontend/src/pages/Admin/requestErrors.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import * as React from 'react';
+import {
+  List,
+  Datagrid,
+  TextField,
+  DateField,
+  Show,
+  SimpleShowLayout,
+  Edit,
+  TextInput,
+  SimpleForm,
+  DateInput,
+  TopToolbar,
+  ListButton,
+} from 'react-admin';
+
+// eslint-disable-next-line react/prop-types
+const RequestErrorShowActions = ({ basePath }) => (
+  <TopToolbar>
+    <ListButton basePath={basePath} />
+  </TopToolbar>
+);
+
+const RequestErrorList = (props) => (
+  <List {...props}>
+    <Datagrid rowClick="edit">
+      <TextField source="id" />
+      <TextField source="operation" />
+      <TextField source="uri" />
+      <TextField source="method" />
+      <TextField source="requestBody" />
+      <TextField source="responseBody" />
```

Comment:
`requestBody` and `responseBody` are both really huge. I think it might be good to not display them in the list view and instead just on the show view.

Reply:
Removed these 2 from the list display.

Reply:
And added horizontal scroll bar.